src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1121,4 +1121,13 @@ export function clinicalDataCountComparator(a: ClinicalDataCount, b: ClinicalDat
     else {
         return b.count - a.count;
     }
+}
+
+export function submitToPage(url:string, params: { [id: string]: string }, target?: string) {
+    try {
+        window.localStorage.setItem("legacyStudySubmission", JSON.stringify(params))
+    } catch (e) {
+        window.localStorage.clear();
```

Comment:
this doesn't try to set the submission again after clearing the localStorage!??

Reply:
the whole point is the clear localstorage so we can set it.  if we just submit then what are we doing?????